OpenGoogleSignInSDK.podspec

```diff
@@ -0,0 +1,20 @@
+
+Pod::Spec.new do |s|
+  s.name = "OpenGoogleSignInSDK"
+  s.version = "1.0.0"
```

Comment:
If we have version 1.1.0 it should probably be also in here right?

Reply:
nice catch! Thank you! Without it tapestry won't work. 👍